S/SARSOP/build_tarballs.jl

```diff
@@ -0,0 +1,55 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "SARSOP"
+version = v"0.96.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/JuliaPOMDP/sarsop.git", "903fa55df637d127c1068700377995ee395475a6")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+cd sarsop/src
+make -j${nprocs} CC=${CC} CXX=${CXX} LINKER=${CXX} DESTDIR=${prefix}
+make install DESTDIR=${prefix}
+if [[ "${target}" == *mingw* ]]; then
+    for file in ${prefix}/bin/*;
+    do
+        mv $file $file.exe
+    done
+fi
+install_license ${WORKSPACE}/srcdir/sarsop/license/*
+exit
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = [
+    Linux(:i686, libc=:glibc),
+    Linux(:x86_64, libc=:glibc),
+    MacOS(:x86_64),
+    Windows(:i686),
+    Windows(:x86_64)
```

Comment:
Ok, let's do this for the time being, then we'll see what's going on
```suggestion
    Platform("i686", "linux"; libc="glibc"),
    Platform("x86_64", "linux"; libc="glibc"),
    Platform("x86_64", "macos"),
    Platform("i686", "windows"),
    Platform("x86_64", "windows"),
```